mne_features/feature_extraction.py

```diff
@@ -64,6 +69,40 @@ def get_feature_names(self):
         else:
             return np.arange(self.output_shape_).astype(str)
 
+    def get_params(self, deep=True):
+        """ Get the parameters (if any) of the given feature function.
+
+        Parameters
+        ----------
+        deep : bool (default: True)
+            If True, the method will get the parameters of the transformer and
+            subobjects. (See `sklearn.preprocessing.FunctionTransformer`).
+        """
+        _params = super(FeatureFunctionTransformer, self).get_params(deep=deep)
+        if hasattr(_params['func'], 'func'):
+            # If `_params['func'] is of type `functools.partial`
+            _to_inspect = _params['func'].func
+        elif hasattr(_params['func'], 'py_func'):
+            # If `_params['func'] is a jitted Python function
+            _to_inspect = _params['func'].py_func
+        else:
+            # If `_params['func'] is an actual Python function
+            _to_inspect = _params['func']
+        args, _, _ = getargs(_to_inspect.func_code)
+        defaults = _to_inspect.func_defaults
+        if defaults is None:
+            return dict()
+        else:
+            n_defaults = len(defaults)
+            func_params = {key: value for key, value in
+                           zip(args[-n_defaults:], defaults)}
```

Comment:
get_params should use self.kw_args and overwrite the inspected parameters if present in self.kw_args

Reply:
Instead of get_params, set_params should do that, shouldn't it ?

Reply:
what should set_params do?

Reply:
Given a dict (`param: value`) it should set the params of the function in the transformer (kw_args).